### ISSUE TITLE
[hackerone] 313440

### ISSUE THREAD

**Issue by jumde** [brave/browser-laptop#13088]
https://hackerone.com/bugs?report_id=313440

**Comment by jumde** [comment]
Verification Steps:

- Open - https://nodejs.org/dist/v8.9.4/node-v8.9.4.pkg
- Run -

> $ xattr ~/Downloads/node-v8.9.4.pkg
> com.apple.metadata:kMDItemWhereFroms
> com.apple.quarantine
> 

Quarantine flag should be enabled for the file

- Check if brave updates are functional

**Comment by alexwykoff** [comment]
@diracdeltas @jumde is this release blocking?

**Comment by diracdeltas** [comment]
if possible

**Comment by diracdeltas** [comment]
@bsclifton yup, the muon changed will fix the issue. It's opened here so QA has steps to verify it. (In particular, they should verify that the change does not break brave autoupdate)

**Comment by bsclifton** [comment]
Moving to `0.22.x`; we're going to have `0.21.x-C65` only contain the Chromium upgrade 😄 👍

**Comment by LaurenWags** [comment]
Verified on macOS 10.12.6 x64 using the following build:
* 0.22.7 https://github.com/brave/browser-laptop/commit/8bb7e77f69bca186429aafda4bc6ba4d82a5de47
* libchromiumcontent: 65.0.3325.181
* muon: 5.1.1

**Comment by kjozwiak** [comment]
Specific to `mac` and the gatekeeper.

**Comment by Liunkae** [comment]
+1 from me (#13085). Thanks for fixing!

**Comment by bsclifton** [comment]
Reverted to fix https://github.com/brave/browser-laptop/issues/13817 with https://github.com/brave/muon/pull/570 in Muon and https://github.com/brave/browser-laptop/pull/13819 on browser-laptop
